python/lsst/ap/association/filterDiaSourceCatalog.py

```diff
@@ -104,6 +104,19 @@ class FilterDiaSourceCatalogConfig(
         "sky sources."
     )
 
+    doRemoveNegativeDirectImageSources = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc="Remove DIASources with negative scienceFlux/scienceFluxErr "
+        "according to a configurable threshold.",
+    )
+
+    min_allowed_direct_snr = pexConfig.Field(
```

Comment:
The name of this config should match the format of the other configs.
```suggestion
    minAllowedDirectSnr = pexConfig.Field(
```